api/src/controllers/support-center.js

```diff
@@ -147,13 +147,13 @@ router.put("/ticket/:id", passport.authenticate(["referent", "young"], { session
   }
 });
 
-// Create a new ticket
+// Create a new ticket while authenticated
 router.post("/ticket", passport.authenticate(["referent", "young"], { session: false, failWithError: true }), async (req, res) => {
   try {
-    const { subject, type, message, tags, title } = req.body;
-    const email = req.user.email;
```

Comment:
pourquoi changer ca ?

il y a le passport -> il y a forément un `req.user`

Reply:
C'est rectifié dans le commit suivant :)

Reply:
c'est a dire ?